app/controllers/state_file/questions/retirement_income_subtraction_controller.rb

```diff
@@ -15,28 +19,28 @@ def initialized_update_form
         form_class.new(find_or_create_state_specific_followup, form_params)
       end
 
-      def self.show?(intake)
-        Flipper.enabled?(:show_retirement_ui) && intake.eligible_1099rs.present?
-      end
-
       private
 
       def num_items
         current_intake.eligible_1099rs.count
       end
 
       def load_item(index)
-        @state_file_1099r = current_intake.eligible_1099rs[index]
-        if @state_file_1099r.nil?
+        @state_file_1099r = self.class.load_1099r(current_intake, index)
+        if state_file_1099r.nil?
           render "public_pages/page_not_found", status: 404
         end
       end
 
+      def self.load_1099r(intake, index)
+        intake.eligible_1099rs[index]
+      end
```

Comment:
why is this a class method?

Reply:
so that it can be called by `self.show?`, which (in the case of `IdIneligibleRetirementAndPensionIncomeController`) needs to grab the correct 1099-R to determine whether the associated followup indicates that we should show the page.